engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -1515,6 +1515,32 @@ namespace dmGameSystem
         }
     }
 
+    int GetVelocityIteration(void* _world)
+    {
+        CollisionWorld* world = (CollisionWorld*)_world;
+        if (world->m_3D)
+        {
+            return dmPhysics::GetVelocityIteration(world->m_World2D);//0;
```

Comment:
You mustn't call a 2D function when only the 3D context is set!

Reply:
yeah, I just test , will remove or replace with calling to get Bullet's iteration instead.